app/particle/particle_simulation.py

```diff
@@ -189,8 +245,8 @@ def do_GET(self):
             self.send_header("Access-Control-Allow-Origin", "*")
             self.end_headers()
             self.wfile.write(message.encode("utf-8"))
-        else:
-            util.info("GET sent to " + str(parsed_path[2]))
+#        else:
+#            util.info("GET sent to " + str(parsed_path[2]))
```

Comment:
Code that is commented out can be removed; everything should be in version control should the desire to revert changes arise.